### DISCUSSION TITLE
Adjusting column width?

### DISCUSSION THREAD

**Discussion by Fluxus00** [inventree/InvenTree#4628] [Q&A] [answered]
Is it possible to adjust column width for each user? For example can user 1 adjust the name column so it has a width of 10 and user 2 so it has a width  of 15?  (Don't mind the measurement Unit)

**Comment by SchrodingersGat** [comment]
You're going to have to provide some more context here please. Perhaps a screenshot. What columns?

**Comment by Fluxus00** [comment]
![image](https://user-images.githubusercontent.com/115626396/232721462-0f67ab50-3123-45b7-b106-5e114124cfce.png)
Those for example in the parts overview. Let us take the description column, is it possible to manually adjust its width?

**Comment by SchrodingersGat** [comment, marked as answer]
@Fluxus00 might be technically possible. We use [bootstrap-table](https://bootstrap-table.com) for table rendering, and you can manually adjust column widths:

https://bootstrap-table.com/docs/api/column-options/#width

However we'd need to be able to set column widths dynamically for each column, and store these settings in the browser cache (no communication back to the server). It would then be fixed to a particular browser on a particular device.

If you are interested in developing this I can provide some pointers, but this would not be a high priority development target for me.

**Reply by Fluxus00** [reply]
@SchrodingersGat Sorry but I am not confident enough in my own skills to provide something useful in a reasonable time. It was just a question, but thank you anyways for your support.
